spec/models/user_spec.rb

```diff
@@ -151,30 +206,31 @@
       webauth_privgroup_str = 'sdr:administrator-role|sdr:manager-role|sdr:viewer-role'
       mock_webauth = double('webauth', :login => 'asdf', :logged_in? => true, :privgroup => webauth_privgroup_str)
       user = User.find_or_create_by_webauth(mock_webauth)
-      expect(user.is_admin  ).to be_truthy
-      expect(user.is_manager).to be_truthy
-      expect(user.is_viewer ).to be_truthy
-      user.set_groups_to_impersonate(['workgroup:sdr:not-an-administrator-role', 'workgroup:sdr:not-a-manager-role', 'workgroup:sdr:not-a-viewer-role'])
-      expect(user.is_admin  ).to be_falsey
-      expect(user.is_manager).to be_falsey
-      expect(user.is_viewer ).to be_falsey
+      expect(user.is_admin).to be true
```

Comment:
So it seems like we have gone back in forth between `be_truthy` and `be true` and `eq true` a few times. Could we discuss the reasons for the historical and current changes?

@atz 

https://github.com/sul-dlss/argo/commit/bc56d4039d37ade6cc2aa86b9c7cba2cd03164be